src/store/storage/local.ts

```diff
@@ -32,8 +32,12 @@ export class Local {
         }
 
         return {
-            keymaps: config.keymaps,
-            macros: config.macros,
+            keymaps: {
```

Comment:
Why does it need to be wrapped in the object? I thought we have already talked about this and it worked with arrays perfectly.